### ISSUE TITLE
Include/exclude categories from notification summaries

### ISSUE THREAD

**Issue by richardkriesman** [8bitPit/Niagara-Issues#1685]
**Is your feature request related to a problem? Please describe.**

Some apps, like Discord, send notifications that are both important but non-urgent (e.g. messages posted in a server channel) and important and urgent (e.g. direct messages). The latter isn't classified as a conversation in Android, so when using notification summaries, they are not excluded from notifications summaries.

**Describe the solution you'd like**

Ideally, I'd like some way to select specific notification categories for apps in the summary, either as a whitelist or blacklist (I think I'd personally prefer a blacklist, but a whitelist may be clearer to the user).

**Comment by Maxr1998** [comment]
I agree that this would make sense. More granular controls over notifications would definitely make this feature more useful, but might also confuse more inexperienced users. Finding a method to implement this in a user-friendly way might take some time and will require thorough planning.

**Comment by Ceddix** [comment]
> I agree that this would make sense. More granular controls over notifications would definitely make this feature more useful, but might also confuse more inexperienced users. Finding a method to implement this in a user-friendly way might take some time and will require thorough planning.

Probably you could add a way to expand an app inside the settings page.

For example if you swipe right on Discord it shows all the possible notification categories.

**Comment by imericxu** [comment]
To add to this, it would be great if we could toggle individual notification channels (which is already a native Android feature) for the Notification Summary. I don't always want to just flat out ignore all notifications from an app.
